TelepresenceBot/server-unit/core/server.js

```diff
@@ -36,11 +36,13 @@ io.sockets.on('connection', function (client) {
     switch(clientType) {
         case ClientType.HUMAN:
             humans.push(client.id);
+            leaveAllRooms(client);
```

Comment:
By default, all clients are in their own room matching their client id. To ship quickly I am limiting us to one room for testing purposes, hence removing them from all rooms and adding them to the only room 'London'. Practically this means that we only have one `bot` and one `human` capable of connecting atm.

In a follow-up PR I'll actually iterate through all of the bots, checking their rooms to see if they are connected with a human and if not allowing the connection to go ahead with a new human.